diff -rc server/api/src/rsGenQuery.c server_updated/api/src/rsGenQuery.c
*** server/api/src/rsGenQuery.c	Fri Dec 17 09:08:32 2010
--- server_updated/api/src/rsGenQuery.c	Fri Dec 17 09:08:59 2010
***************
*** 85,93 ****
  #if 0
         msParam_t *outMsParam;
  #endif
!        memset((char*)&rei,0,sizeof(rei));
!        status = applyRule ("acAclPolicy", NULL, &rei, NO_SAVE_REI);
!        ruleResult = rei.status;
         if (status==0) {
  	  ruleExecuted=1; /* No need to retry next time since it
                               succeeded.  Since this is called at
--- 85,97 ----
  #if 0
         msParam_t *outMsParam;
  #endif
! //       memset((char*)&rei,0,sizeof(rei));
! //       status = applyRule ("acAclPolicy", NULL, &rei, NO_SAVE_REI);
! //       ruleResult = rei.status;
! // SDF - allow the acAclPolicy rule access to the user name
! // NB: irods login only as GSI et al do not know the username at this point
!        status = applyRule ("acAclPolicy", NULL, &rei2, NO_SAVE_REI);
!        ruleResult = rei2.status;
         if (status==0) {
  	  ruleExecuted=1; /* No need to retry next time since it
                               succeeded.  Since this is called at
